lib/tasks/bulk.rake

```diff
@@ -1,6 +1,17 @@
 # frozen_string_literal: true
 namespace :figgy do
   namespace :bulk do
+    desc "Move a box to a new project"
+    task move_box: :environment do
+      target_project_id = ENV["TARGET_PROJECT_ID"]
```

Comment:
I may be misinterpreting this, but I'm curious if this meens we need to add environment variables each time before this is run for the target project and the box? Would it make more sense to specify them as arguments that can be added onto the rake command?

Reply:
Arguments in rake commands are pretty tough, so we use env variables in the command line as a replacement.